Updated Argonauts API integration to fix crashing

Siriuo opened the pull request:
I am not going to pretend to be a professional in Java at all, however I have been running Argonauts (Now Odyssey Allies) and along with Modern Industrialization and have been experiencing crashing due to some changes that have been made to their GuildAPI class.

`java.lang.NoClassDefFoundError: earth/terrarium/argonauts/api/guild/GuildApi`

Anyway, I have made a fix for it which appears to work during my testing.

Siriuo committed:
Updated Argonauts API integration to fix crashing

Siriuo committed:
Formatting Fixes

Technici4n commented on src/main/java/aztech/modern_industrialization/compat/argonauts/ArgonautsFacadeImpl.java:
```diff
@@ -33,14 +33,16 @@
     public Collection<UUID> getOtherPlayersInGuild(MinecraftServer server, UUID playerUuid) {
-        var guild = GuildApi.API.getPlayerGuild(server, playerUuid);
-        if (guild == null) {
+        var player = server.getPlayerList().getPlayer(playerUuid);
```
This should not use the `Player` explicitly because it also has to work when the player referenced by `playerUuid` is offline.

Siriuo replied:
Hope this is better.

Siriuo committed:
Ensure playerUuid is referenced
Passing the level directly to argonauts api instead as this is now requiring a level instead of the server

Technici4n commented on src/main/java/aztech/modern_industrialization/compat/argonauts/ArgonautsFacadeImpl.java:
```diff
@@ -23,24 +23,25 @@
+        for (var member : guild.get().members().entrySet()) {
+
+            if (!member.getKey().equals(playerUuid)) {
+                out.add(member.getKey());
```
```suggestion
        for (var memberUuid : guild.get().members().keySet()) {
            if (!memberUuid.equals(playerUuid)) {
                out.add(memberUuid);
```

Technici4n commented:
Looks good, just one change please.

Siriuo committed:
Change member to memberUuid and to get the keySet instead of entrySet

Siriuo commented:
Great.... Certainly all applied

Technici4n commented:
Thanks!
